hw/dv/tools/dvsim/questa.hjson

```diff
@@ -0,0 +1,111 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "vlib work && {job_prefix} vlog"
+  run_cmd:    "{job_prefix} vsim"
+
+
+  build_opts: ["-sv",
+               "-timescale 1ns/1ps",
+               "+incdir+{QUESTA_HOME}/verilog_src/uvm-1.2/src/",
+               "{QUESTA_HOME}/verilog_src/uvm-1.2/src/uvm_pkg.sv",
+               // Multi-file compilation unit option for Questa
+               // Needed to use macros across CU-s
+               "-mfcu"
+               "-f {sv_flist}"]
+
+  run_opts:   ["-sv_seed {seed}",
+               // List multiple tops for the simulation
+               "{sim_tops}",
+               // dv_macros.svh has a macro printing null using %0d
+               // format specifier, Questa throws an error on this 
+               // we demote this error in Questa
+               "-suppress vsim-8323",
```

Comment:
There's still an outstanding review request on this code at https://github.com/lowRISC/opentitan/pull/4616#discussion_r545886589

Reply:
@imphil - do you refer to your previous detailed notes on rebase or the issue around %d on _null_ as in

https://github.com/lowRISC/opentitan/pull/4616#discussion_r545886589

On the rebase part, I will take care from next PR onwards.

On the %d stuff - I believe changes will be in atleast 2 files and we need to rerun on all tools as well

Reply:
I will suggest dealing with it in a separate PR.

Reply:
Sure will do